Apps/Sandcastle/gallery/Shared Context.html

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
```

Comment:
Is there a way to hide a Sandcastle from the public gallery? If not, should I remove this one that illustrates the use of temporarily `@private` APIs?

Reply:
Yes, you can move the html file to the `gallery/development` directory to hide the example from the public gallery. Examples in this directory are only available when running CesiumJS/Sandcastle in local development.